### PR TITLE
Fixed some aria violations

### PR DESCRIPTION
**Pull request by Stanzilla**
#### Database Migration
NO

#### Description

Fixes a few aria violations that I found when running lighthouse, the remaining one is in 3rd party code at https://github.com/moroshko/react-autosuggest/pull/830

### CODE REVIEW

File: frontend/src/Components/Link/IconButton.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ function IconButton(props) {
         className,
         isDisabled && styles.isDisabled
       )}
+      name={name}
```
Comment (markus101) [inline frontend/src/Components/Link/IconButton.js:26]: `name` is an object, I don't think we want to pass that as the name attribute on the button

File: frontend/src/Components/DescriptionList/DescriptionListItem.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ class DescriptionListItem extends Component {
     } = this.props;
 
     return (
-      <span className={className}>
+      <div className={className}>
```
Comment (markus101) [inline frontend/src/Components/DescriptionList/DescriptionListItem.js:21]: aria related or something else?

Comment (Stanzilla) [reply frontend/src/Components/DescriptionList/DescriptionListItem.js:21]: HTML spec actually, span are not valid inside of there

File: frontend/src/Components/Link/IconButton.js (hunk 1 above)
Comment (Stanzilla) [reply frontend/src/Components/Link/IconButton.js]: Fixed

**Pull request merged**